extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -2033,6 +2033,18 @@ describe('amp-a4a', () => {
     });
   });
 
+  describe('shouldPreferentialRenderWithoutCrypto', () => {
+    it('returns false by default', () => {
+      return createIframePromise().then(fixture => {
+        setupForAdTesting(fixture);
+        const doc = fixture.doc;
+        const a4aElement = createA4aElement(doc);
+        const a4a = new AmpA4A(a4aElement);
+        expect(a4a.shouldPreferentialRenderWithoutCrypto()).to.be.false;
```

Comment:
Need more test coverage than this.  Please add the following tests:

- executes the promise chain without crypto support, default shouldPreferentialRenderWithoutCrypto, but with crypto key in response: should not get preferential render
- identical but shouldPreferentialRenderWithoutCrypto is true and we do get preferential render
- shouldPreferentialRenderWithoutCrypto is true but no crypto header and verify no preferential render.

Reply:
done